### DISCUSSION TITLE
Progress?

### DISCUSSION THREAD

**Discussion by stefonarch** [qtilities/sddm-conf#3] [General]
Any news here? @tsujan ?

**Comment by tsujan** [comment]
I'm afraid, not. I knew I would be busy but I underestimated the amount and diversity of codes that needed my attention, both in LXQt and in my own projects. Apparently, I should accept that I've reached the limits of my coding time.

Since, realistically speaking, I won't have time to read sddm-conf's code soon, and because you and @redtide have tested it, I suggest its release. It'll be a great help to sddm users — especially LXQt users — who don't want to install KDE stuff. In addition, the more users, the more feedback, which will be important for its development.

Sorry, I should have said that sooner but, each day, I thought I might find some time, which I didn't :(

**Comment by redtide** [comment]
I'm sorry, although I said the changes should not be difficult to make; it's only about the current saving method which is not correct, but I'm still in a condition that i can't do anything (yet).

**Comment by tsujan** [comment]
Any news? Do you have a plan to publish it?

For some reason, GitHub shows that there was a commit 8 days ago.

**Reply by stefonarch** [reply]
In fact, but it went over email. Still about the config files, It works fine if present one of `/etc/sddm.conf` or `/etc/sddm.d/sddm.conf `.
But I saw when no config file is present it saves in nowhere, doesn't create any config file.
Should open an issue about.

**Reply by redtide** [reply]
I force-pushed over the first commit so it appears as recent. I'm in a very difficult times and not focus at all in any programming stuff, so I don't want to publish something that will fail with no chance to give support, that's why I was asking for some help.

**Comment by redtide** [comment]
I published an [AUR](https://aur.archlinux.org/packages/sddm-conf-git) package after making this public.

**Reply by tsujan** [reply]
Although I don't change SDDM settings, I installed it in case I need to. Who needs `systemsettings5` for that anymore? Nice app!

![sddm-config](https://user-images.githubusercontent.com/981076/163839314-23ff2582-9493-42a4-be8d-c88673203c88.png)

**Reply by stefonarch** [reply]
> I published an [AUR](https://aur.archlinux.org/packages/sddm-conf-git) package after making this public.

Nice, as Tsujan said: user feedback is needed. I suggest also publishing a 0.1 release.

**Reply by redtide** [reply]
> Nice, as Tsujan said: user feedback is needed. I suggest also publishing a 0.1 release.

I thought the same thing while looking at the PKGBUILD but if it's OK I would wait a bit for some other feedback before tagging a release

**Reply by tsujan** [reply]
> I would wait a bit for some other feedback before tagging a release

And I might bother you with more PRs.

**Reply by redtide** [reply]
I think you could even commit directly, in the end it's a simple GUI, so no much things to discuss about it, no?

**Reply by tsujan** [reply]
I'll request your review when making PRs. If there's something to test, I'll request @stefonarch's review too; he performs the best tests.

**Reply by stefonarch** [reply]
This makes me think: it should be shown under "System Settings" as with
`Categories=Settings;DesktopSettings;Qt;System;` but it's maybe the  filtering which could be improved,

**Reply by tsujan** [reply]
> Categories=Settings;DesktopSettings;Qt;System;

Or add `LXQt;` to it and it'll be under LXQt Settings. I did so for Kvantum years ago, because I wanted LXQt users to know that it's tuned to LXQt too.

![sddm](https://user-images.githubusercontent.com/981076/163995754-083901d1-4e6d-4069-9b55-c534b929b1ed.png)

**Comment by redtide** [comment]
I'm not using much this app, I set the manager once for all usually.
I just see it as a generic app configurator (similar to Picom-conf, despite the stripped down LXQt Tools used only for translations) to be usable under any DE, so I'm not sure about moving it under the LXQt category; not even about the System settings one, I just like to see them appear directly on the Preferences menu as is, which is what I use instead the Config Center.
I may be wrong or I'm missing something (like some different menu/category management) and this is not a "vital" thing, so feel free to change it.

**Reply by tsujan** [reply]
Even when I was a KDE user years ago, I didn't change SDDM settings because it might be broken after a future update.

The point is that you've made an app that didn't exist before: An SDDM configuration utility that doesn't have artificial KDE dependencies. KDE devs like to add KDE dependencies where there's no need to them.

**Reply by redtide** [reply]
I noticed that, which is more or less why I removed almost immediately KWin after a try in LXQt. This app is super simple, it just saves changes to a text file, I can't imagine where dependencies are needed, except the polkit for admin rights.
If I'm not wrong also sddm-config-editor have about the same dependencies, just using QML instead.

**Reply by tsujan** [reply]
I have a full KDE installation (for testing my apps under it); so, KDE dependencies aren't a problem for me. But I don't like false dependencies in any app.

BTW, I hate QML ;)
